### PR TITLE
Minor updates to Ubernetes phase 1 design doc.

### PR DESCRIPTION
**Pull request by ghost**
This change is [Reviewable](http://reviewable.k8s.io/reviews/kubernetes/kubernetes/23846)

### CODE REVIEW

**Comment by nikhiljindal** [comment]
cc @kubernetes/sig-cluster-federation

File: docs/design/federation-phase-1.md [hunk 1]
```diff
@@ -157,23 +168,42 @@ clusters over AWS clusters" or "spread load equally across clusters").
...
+   persistent store of desired state (to cater for, e.g, the
+   possibility of some external agent deleting or modifying the
+   underlying resources).  Note that it will be necessary to be able
+   to disable this behaviour to make it possible for things like
+   Kubernetes pod autoscalers, which intentionally and automatically
+   modify Kubernetes replication controller's desired state, to
+   interoperate correctly with Ubernetes.  The details of this
+   interoperability are as yet undefined. In practise 1 and 2 are
+   implemented as a single reconciliation loop.
+1. It periodically retrieves available resource metrics from
```
Comment (huangyuqi) [inline docs/design/federation-phase-1.md]: Hi, @quinton-hoole ,
from https://github.com/kubernetes/kubernetes/issues/23614#issuecomment-204148122. 
I got that , in phase 1, do we just need get the status of underlying Kubernetes clusters ,and choose a cluster randomly?
BTW, shall we use the restapi `https://kubernetes/healthz` to get the status of cluster?

File: docs/design/federation-phase-1.md [hunk 2]
```diff
@@ -140,14 +140,25 @@ time.
...
-and post them back to `etcd` storage.
+Whereas the Kubernetes scheduler schedules pending pods onto
+underlying nodes, the Ubernetes Scheduler schedules resources onto the
+underlying Kubernetes clusters.  So the Ubernetes scheduler creates
+per-cluster Kubernetes API objects corresponding to Ubernetes API
+objects.
+
+For example, for Replication Controllers:
+
+1. watches it's persistent store (typically etcd) for unscheduled
```
Comment (nikhiljindal) [inline docs/design/federation-phase-1.md]: To be clear, the controller should not talk to etcd directly. It should go through the api server.

File: docs/design/federation-phase-1.md [hunk 3]
```diff
@@ -140,14 +140,25 @@ time.
...
+Ubernetes replication controllers (those that have not yet been
+scheduled onto underlying Kubernetes clusters)
+1. performs the global scheduling work, by consulting the policy
+engine to decide how to spit workloads among clusters
+1. creates a local record of a Kubernetes Replication Controller for 
+each underlying cluster onto which it decides to schedule (including
+the desired number of replicas for each cluster).  This constitutes
+the desired state of each Kubernetes replication controller, from the point of view
+of Ubernetes.
+1. saves these records back to the Ubernetes persistent store.
```
Comment (nikhiljindal) [inline docs/design/federation-phase-1.md]: again via the ubernetes api server.

File: docs/design/federation-phase-1.md [hunk 4]
```diff
@@ -157,23 +168,42 @@ clusters over AWS clusters" or "spread load equally across clusters").
...
-   the Ubernetes control plane.  Which approach is better remains an
-   open topic of discussion.
+The cluster controller's job is to reconcile the desired state of the
+underlying Kubernetes clusters (as recorded by Ubernetes in it's
+persistent store of desired objects) against the real Kubernetes
+clusters' states.
+
+To achieve this, it performs the following three kinds of work:
+
+1. It watches the Ubernetes persistent store for new or changed
```
Comment (nikhiljindal) [inline docs/design/federation-phase-1.md]: Do we want to have a single controller that watches all resources? or a controller for every resource? (like we will have ServiceController and ReplicaSetController only for now). Apart from creating or modifying the corresponding k8s resources, these controller will also set appropriate fields in the u8s resources (for ex: ReplicaSetController will set the ObservedGeneration for ReplicaSet, when it observes a replicaset update)

**Pull request closed without merging**